config/locales/en.yml

```diff
@@ -267,6 +266,44 @@ en:
+        part9: "If you would like Open Doors to update the redirect to point to your pre-existing work, please delete the imported copy, and contact Open Doors at %{open_doors_link} with your AO3 account name, your account name on the imported archive, and the title and URL of the fanwork you would like the redirect to point to. (If you have multiple works you would like to change the redirects for, you can list these in one email.)"
+        part10: "If you had other works on the imported archive under an e-mail address you can no longer access, please contact Open Doors with any information that can help verify your identity."
+        part11: "For other inquiries, please contact AO3 Support at %{support_link}."
+        part13: |-
+          Best,
+          The Open Doors team
+          Organization for Transformative Works
+      html:
+        part1_html: "You're receiving this e-mail because an archive has recently been imported by %{open_doors_link} into the %{app_link} (%{app_short_name}), and we believe that the following fanworks belong to you. We'd like to give you a chance to claim (or delete/orphan) these works if you want to. And if you don't already have an account under a different e-mail, we'd like to invite you aboard!"
+        open_doors: "Open Doors"
+        part2_html: "You can read announcements about recent archive moves at %{news_link}, and find additional information on Open Doors' %{open_doors_faq_link} or %{open_doors_tutorial_link}. For any questions not answered in the FAQ, tutorials, or this e-mail, please %{support_link}."
+        faq_page: "FAQ page"
+        tutorial_page: "tutorial page"
+        contact_support: "contact AO3 support"
+        part3_html: "If this is a mistake and these are not your works, please don't delete them! Please just %{open_doors_link} and we will sort it out."
+        contact_open_doors: "contact Open Doors"
+        part4_html: "Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please %{support_link}."
+        part7_html: "If these works do belong to you, but you don't want them, you can orphan (so that they remain on the AO3, but with your name removed) or delete them (so that they are entirely removed from the AO3). You do not need to add these works to any account in order to orphan or delete them--you can do this directly from the claim link above. (For assistance, please %{support_link} %>.)"
```

Comment:
There's a `%>` that shouldn't be at the end of this line: `%{support_link} %>.)"`